src/jit/optimizer.cpp

```diff
@@ -7837,7 +7837,9 @@ GenTree *           Compiler::optIsBoolCond(GenTree *   condBranch,
     if  (opr2->gtOper != GT_CNS_INT)
         return  NULL;
 
```

Comment:
Rather than waiting for a wide-spread CC conforming change, JIt team has been making CC conforming changes around or to the existing code that they are touching.  This is not a must do though and left to individuals.